azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -11,6 +11,7 @@ package AzureFunctionsRpcMessages;
 
 import "google/protobuf/duration.proto";
 import "identity/ClaimsIdentityRpc.proto";
+import "shared/NullableTypes.proto";
```

Comment:
Get the latest proto: https://github.com/Azure/azure-functions-language-worker-protobuf/releases/tag/v1.2.0-protofile